app/parsers/facet_parser.rb

```diff
@@ -1,12 +1,16 @@
 module FacetParser
   def self.parse(facet)
-    case facet.type
-    when 'multi-select', 'text'
-      SelectFacet.new(facet)
-    when 'date'
-      DateFacet.new(facet)
+    if facet.filterable
+      case facet.type
+      when 'text'
+        SelectFacet.new(facet)
+      when 'date'
+        DateFacet.new(facet)
+      else
+        raise ArgumentError.new("Unknown filterable facet type: #{facet.type}")
+      end
     else
-      raise ArgumentError.new("Unknown facet type: #{facet.type}")
+      Facet.new(facet)
```

Comment:
Does it matter than non-filterable facets will just be an instance of `Facet`? Is there a difference in behaviour between a text facet and a date facet when it comes to displaying the metadata on results?